locales/templates/app.pot

```diff
@@ -1556,77 +1556,617 @@ msgid ""
+msgstr ""
+
+#. Shopping Search Input Placeholder
+msgctxt "shopping_search_placeholder"
+msgid "Type the product you want to search"
+msgstr ""
+
+#. Shopping Search Prompt Message
+msgctxt "shopping_search_prompt_message_body"
+msgid "Compare prices across many online stores with Smart Shopping Search."
+msgstr ""
+
+msgctxt "shopping_search_prompt_message_button"
+msgid "Search"
+msgstr ""
+
+#. Switch UI Onboarding
+msgctxt "shopping_switch_ui_onboarding_body"
 msgid ""
-"The 7-day free VPN service is ready for you. Install now to browse more "
-"privately and securely."
+"Tap the shopping sites below to browse more results, or tap this button "
+"to edit."
+msgstr ""
+
+msgctxt "shopping_switch_ui_onboarding_button"
+msgid "OK"
+msgstr ""
+
+#. Artboard Name: 2.2 Switch UI Preference Behaviors
+#. Switch UI Preferences
+msgctxt "shopping_switch_ui_preferences_title"
+msgid "Shopping Search Settings"
+msgstr ""
+
+msgctxt "shopping_switch_ui_preferences_subtitle"
+msgid "Sites to compare"
 msgstr ""
 
-#. Message shown in Home Screen with an icon. Users can choose either they
-#. want this feature or not. We'll evaluate if we want to implement this
-#. feature base on the users' feedback.
-msgctxt "exp_survey_wifi"
+#. Artboard Name: 3.1 Switch UI Navigation - Activity Switch
+#. Switch UI Tab Tray. %1$s is the keyword user input
+#, c-format
+msgctxt "shopping_switch_ui_tabtray_title"
+msgid "Shopping Search: “%1$s”"
+msgstr ""
+
+msgctxt "shopping_switch_ui_tabtray_title_no_keyword"
+msgid "Shopping Search"
+msgstr ""
+
+#. Artboard Name: 3.3 Switch UI Navigation - Tab Tray
+#. Closing dialog content.
+msgctxt "shopping_closing_dialog_close"
+msgid "Close All"
+msgstr ""
+
+msgctxt "shopping_closing_dialog_cancel"
+msgid "Cancel"
+msgstr ""
+
+msgctxt "shopping_closing_dialog_body"
+msgid "Close all shopping search results?"
+msgstr ""
+
+#. Artboard Name: 4.1 Shopping Deals Overview
+#. Shopping Home
+msgctxt "shopping_vertical_title"
+msgid "Shopping"
+msgstr ""
+
+msgctxt "shopping_vertical_category_1"
+msgid "Deals"
+msgstr ""
+
+msgctxt "shopping_vertical_category_2"
+msgid "Coupons"
+msgstr ""
+
+msgctxt "shopping_vertical_category_3"
+msgid "Vouchers"
+msgstr ""
+
+#. Shopping Deals
+msgctxt "shopping_deal_subcategory_1"
+msgid "Top Rated"
+msgstr ""
+
+msgctxt "shopping_deal_subcategory_2"
+msgid "Best Sellers"
+msgstr ""
+
+#. Artboard Name: 5.2 Shopping Coupons Update
+#. Shopping Coupons
+msgctxt "shopping_coupon_expire_zero"
+msgid "Expired"
+msgstr ""
+
+msgctxt "shopping_coupon_expire_one"
+msgid "1 day left"
+msgstr ""
+
+#, c-format
+msgctxt "shopping_coupon_expire_other"
+msgid "%1$d days left"
+msgstr ""
+
+#. Gaming Vertical
+#. Gaming Home
+msgctxt "gaming_vertical_title"
+msgid "Game"
+msgstr ""
+
+msgctxt "gaming_vertical_category_1"
+msgid "Instant play"
+msgstr ""
+
+msgctxt "gaming_vertical_category_2"
+msgid "Download to play"
+msgstr ""
+
+msgctxt "gaming_vertical_new"
+msgid "New"
```

Comment:
This needs a comment, what does this refer to? It could be masculine or feminine in some languages

Reply:
A tag to let users know that it is a new game.